packages/govuk-frontend/tasks/build/package.unit.test.mjs

```diff
@@ -355,6 +355,12 @@ describe('packages/govuk-frontend/dist/', () => {
           )
 
           expect(moduleTextESM).toContain(`export { ${componentClassName} }`)
+
+          // Look for AMD module definition for 'GOVUKFrontend'
+          expect(moduleTextUMD).toContain(
+            "(global = typeof globalThis !== 'undefined' ? globalThis : global || self, factory(global.GOVUKFrontend = global.GOVUKFrontend || {}));"
+          )
```

Comment:
**note** Added the same test as for `all.bundle.js` to ensure the UMD wrapper has the right shape.